docs/blog/september-contributions-showcase.md

```diff
@@ -0,0 +1,120 @@
+
+Thanks to pralkarz, [@benjaminmccann](https://x.com/benjaminmccann) and [@superchupudev](https://x.com/superchupu) for already migrating so many projects to these two libraries!
+
+## tinyexec migrations
+
+The community has been working wonders migrating countless projects from `execa` and other heavyweight libraries to the super lean [tinyexec](https://github.com/tinylibs/tinyexec/).
+
+For example, thanks to pralkarz, [vitest](https://github.com/vitest-dev/vitest) has already moved over!
+
+There are many more PRs currently in progress to do just the same :pray: Great work by the community here.
+
+# Contributions / Improvements
+
+## storybook
+
+The [storybook](https://storybook.js.org/) team have been working closely with the e18e community for some time now, dedicating huge amounts of time to improving the performance and install footprint for everyone.
+
+This month, they've kept up the pace! Just a few examples of what has landed:
+
+- Migrated from `chalk` to `picocolors`
+- Migrated from `fs-extra` to built in `fs` functions
+- Removed `handlebars` and simplified templating internally
+- Removed `prettier` from the core bundle (saved **4MB** on the install size)
+- Replaced `lodash` with `es-toolkit` (more huge savings, especially with tree shaking)
+
+On top of this, the team has actually started a full project focusing on [reducing the install footprint](https://github.com/storybookjs/storybook/issues/29038) in collaboration with the e18e community.
+
+For anyone interested in helping out in the e18e space, this issue a great way to learn some of the techniques used to track down bundle/install size issues.
+
+As a bonus, they even integrated bundle size analysis into their CI so all new PRs can be compared with the main branch. Would love to see that in more projects!
+
+This has all been so great to see and will hopefully lead to other large projects making similar efforts! Definitely some big wins ahead :raised_hands:
+
+## nx
+
+The folks over at [nx](https://github.com/nrwl/nx/) have been incredibly active with the e18e community. They are definitely on the same page and have been moving fast to work with the community to land many improvements.
+
+You can see a lot of this in the [tracking issue](https://github.com/es-tooling/ecosystem-cleanup/issues/117) over on the ecosystem cleanup project. These issues are closing quickly, with contributions coming from a few different people in the e18e space.
+
+Just a few examples of the improvements made:
+
+- Migrating older fs-extra code to use newer, native `fs` functionality
+- Migrating from chalk to picocolors
+- Migrating from glob libraries to fdir
+
+These are all improving the CPU performance of nx and cutting huge subtrees out of the dependency tree at the same time.
+
+## jimp
+
+[jimp](https://github.com/jimp-dev/jimp), an image manipulation library, has dropped an impressive amount of dependencies and reduced their package size hugely.
```

Comment:
This was kind of a wash. They introduced a new version that was a big regression. I noticed it while upgrading and it was a pretty small change to fix it

Reply:
It was jimp #1302. Basically I just removed rspack as an unused dependency plus a few other minor cleanups

Reply:
We should probably rewrite this to instead just mention the size reduction I think then

If I understood correctly 👀